airflow/utils/cli.py

```diff
@@ -328,3 +344,32 @@ def _wrapper(*args, **kwargs):
                     logging.disable(logging.NOTSET)
 
     return cast(T, _wrapper)
+
+
+def get_config_with_source(include_default=False):
```

Comment:
```suggestion
def get_config_with_source(include_default: bool = False) -> str:
```

nit: let's add typehints